packages/grafana-traces/src/IntervalInput/IntervalInput.test.tsx

```diff
@@ -0,0 +1,76 @@
+import { render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
```

Comment:
I think you'll need to add these to the package.json as yarn won't allow resolving to deps in root package.json.